static/js/popovers.js

```diff
@@ -1135,6 +1135,13 @@ exports.hide_all = function () {
     exports.hide_userlist_sidebar();
     stream_popover.hide_streamlist_sidebar();
     exports.hide_all_except_sidebars();
+    $(".dropdown").removeClass("open");
+};
+
+exports.hide_all_popovers = function () {
```

Comment:
It's confusing to have two methods called `popovers.hide_all` and `popovers.hide_all_popovers`.  It's not completely clear when you'd call one and not the other.

Reply:
Will add appropriate comments. And rename the hide_all_popovers to something better. Any suggestion?

Reply:
> Will add appropriate comments, and rename the hide_all_popovers to something better. Any suggestion?

Reply:
the hide_all function has been used in a lot of places. and is difficult to change now, plus I think it fits correctly(hide all elements on the screen) maybe, the hide_all_popovers function can be renamed to hide_only_popovers() ?